atomic-agents/atomic_agents/agents/base_agent.py

```diff
@@ -71,7 +73,10 @@ class BaseAgentConfig(BaseModel):
         0,
         description="Temperature for response generation, typically ranging from 0 to 1.",
     )
-
+    max_tokens: Optional[int] = Field(
+        2048,
```

Comment:
replace `2048` with `None`

This way, we don't enforce any limit by default for other providers.

This will still throw the error if you do not provide the `max_tokens` manually, but that is desired behavior, it _should_ be explicit, like how `instructor` does it.